packages/api/src/types.ts

```diff
@@ -148,6 +148,7 @@ export interface OpenAIPluginManifest {
 export interface ApiKeyAuthInfo {
```

Comment:
Rename the interface -> AuthInfo

Reply:
Will do in another pr since it may affect other files.